LICENSE

```diff
@@ -1,6 +1,8 @@
 MIT License
 
 Copyright (c) 2017-2019 Rafael Beraldo
+Copyright (c) 2017-2024 Walter Tommasi
+Copyright (c) 2023-2024 Kikuo Emoto and Wesley Ford
```

Comment:
@wesdevpro Despite I suggested these changes, I started to contemplate @jtommy's words "it's a community project." A lot more people actually contributed to the project.

Reply:
Resolved internally over discord.